### ISSUE TITLE
Cross-project piping not working with surveys

### ISSUE THREAD

**Issue by glut4** [vanderbilt-redcap/cross-project-piping-module#12]
These page CrossprojectpipingExternalModule.php shows that it is using a direct url to the module which will fail if the user isn't authenticated. The module should be using a noauth url for surveys (and a NOAUTH API url if you want it to work on people who use Shibboleth authentication). (this has been suggest in REDCap community by Andy Martin.

**Comment by mmcev106** [comment]
I don't believe this module was designed/intended to function on surveys.  More security may be warranted before the changes suggested would be safe.  Also, this module has unfortunately not had a support budget for years, and would need someone to commit time or funds before any changes could be made.
